mcs/class/System.Net.Http/System.Net.Http/HttpClient.wasm.cs

```diff
@@ -16,39 +16,16 @@ internal static HttpMessageHandler CreateDefaultHandler()
 
```

Comment:
How about

```cs
using WebAssembly.Net.Http.HttpClient;
```

at the top and then replace this entire body with

```cs
return WasmHttpMessageHandler.GetHttpMessageHandler ();
```

Reply:
Addressed in recent commit